src/clouds.cpp

```diff
@@ -191,7 +193,10 @@ void Clouds::render()
 					(float)p_in_noise_i.X * cloud_size_noise,
 					(float)p_in_noise_i.Y * cloud_size_noise,
 					m_seed, 3, 0.5);
-			grid[i] = (noise >= 0.4);
+			// normalize to 0..1 (given 3 octaves)
+			double noise_bound = 1.0 + 0.5 + 0.25;
```

Comment:
Separate issue:
This should be a 'float', we very rarely use double precision floats in code, and for something as basic as using noise to determine cloud cover this should certainly be a float. I know it was a double before but that was unnecessary.